libs/SalesforceSDK/res/layout/sf__idp_auth_code.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Layout for the auth code activity.
If the user pre-approved the connected app, nothing will be shown in the web view so using a spinner (visible until the web view loads) to make the UX better.